From fde667e889548a064009c45ea4891b78977962b9 Mon Sep 17 00:00:00 2001
From: Damien Le Moal <damien.lemoal@opensource.wdc.com>
Date: Tue, 28 Feb 2023 17:00:32 +0900
Subject: [PATCH 27/34] arm: rk3399-rockpro64: Add interrupts to PCI endpoint
 mode DT node

Add subsystem and client interrupts definitions to the rockchip PCI
eendpoint node.

Signed-off-by: Damien Le Moal <damien.lemoal@opensource.wdc.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index c3489be3852d..92eb541fde9d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -274,6 +274,9 @@ pcie0_ep: pcie-ep@f8000000 {
 			<&cru PCLK_PCIE>, <&cru SCLK_PCIE_PM>;
 		clock-names = "aclk", "aclk-perf",
 				"hclk", "pm";
+		interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH 0>,
+			     <GIC_SPI 51 IRQ_TYPE_LEVEL_HIGH 0>;
+		interrupt-names = "sys", "client";
 		max-functions = /bits/ 8 <8>;
 		num-lanes = <4>;
 		reg = <0x0 0xfd000000 0x0 0x1000000>, <0x0 0xfa000000 0x0 0x2000000>;
-- 
2.39.2

